enterprise/backend/test/metabase_enterprise/advanced_permissions/models/permissions/block_permissions_test.clj

```diff
@@ -106,7 +106,9 @@
               (is (= {:schemas :block}
                      (test-db-perms group-id)))
               (is (= #{(perms/database-block-perms-path (mt/id))}
-                     (db/select-field :object Permissions :group_id group-id))))))))))
+                     (db/select-field :object Permissions {:where [:and
+                                                                   [:like :object "/block/%"]
+                                                                   [:= :group_id group-id]]}))))))))))
```

Comment:
Hmm, I think this change is breaking the purpose of this test?

The test is trying to assert that there _aren't_ data perms, when a group has block perms. But that's not being checked if you're only fetching the block perm path.

You'd have to also include paths like `/db/%`, I think